solr/licenses/raleway-OFL.txt

```diff
@@ -0,0 +1,93 @@
+Copyright 2010 The Raleway Project Authors (impallari@gmail.com), with Reserved Font Name "Raleway".
+
+This Font Software is licensed under the SIL Open Font License, Version 1.1.
+This license is copied below, and is also available with a FAQ at:
+https://openfontlicense.org
+
+
+-----------------------------------------------------------
+SIL OPEN FONT LICENSE Version 1.1 - 26 February 2007

```

Comment:
Any issues with this license in an ASF project?

Reply:
This license is according to the website attribution free under certain circumstances. In our case we bundle it within the web project, and therefore do not have to ship any licensing in the project. I believe that in the long run it is not a bad idea to do it anyway, especially if we have a section that lists other licenses as well.

However, for now I have removed the font files to avoid versioning binary files. It should be fine with built-in fonts for the first versions and we may consider using a library for that in the future.